index.html

```diff
@@ -1472,6 +1472,193 @@ <h2>
         feature/default allowlist=] is [=default allowlist/'self'=].
       </p>
     </section>
+    <section data-cite="webdriver WebDriver-BiDi">
+      <h2>
+        Automation
+      </h2>
+      <p>
+        For the purposes of user-agent automation and application testing, this
+        document defines extensions to the [[[WebDriver]]] and
+        [[[WebDriver-BiDi]]] specifications. It is OPTIONAL for a user agent to
+        support them.
+      </p>
+      <pre class="idl">
+        dictionary GeolocationMockCoordinates {
+          double accuracy = 1;
+          required double latitude;
+          required double longitude;
+          double? altitude = null;
+          double? altitudeAccuracy = null;
+          double? heading = null;
+          double? speed = null;
+        };
+      </pre>
+      <p>
+        To <dfn>set the system coordinates</dfn> given an optional
+        {{GeolocationMockCoordinates}}
+        |coordinates:GeolocationMockCoordinates|, and an optional [=user
+        agent=] |agent|:
```

Comment:
I'm still working on understanding the WebDriver BiDi architecture but in the old WebDriver specification requests have a *session* from which you can derive the *browsing contexts* to which a command will apply. Rather than applying to the entire user agent this allows automation to control only the specific context the developer is targeting.

Reply:
yeah, something like that could work... I'll see how this looks in the implementation and hopefully can base it off that.

Reply:
As mentioned in #146, Chromium already implements this in the Chrome DevTools Protocol and it is scoped to a particular browsing context. This interface is used by Chrome DevTools to provide a UI for setting a mock position. It would be weird if a change made while inspecting one page affected other pages as well. Supporting both page-level and browser-level location overrides would introduce a lot of complexity to the implementation.

Reply:
Agree... that would be weird.